CHANGELOG.md

```diff
@@ -449,7 +449,7 @@ final transaction is created by calling `finish` on the builder.
 #### Changed
 - Simplify the architecture of blockchain traits
 - Improve sync
-- Remove unused varaint HeaderParseFail
+- Remove unused variant HeaderParseFail
```

Comment:
```suggestion
- Remove unused variant `HeaderParseFail`
```
I think that this should be in `code`.

Reply:
Looks like someone else fixed the spelling issue. I added the single quotes, but in general for such an old version I wouldn't normally bother.